docs/about/history.md

```diff
@@ -0,0 +1,190 @@
+> more like "We are Redis", (I even own the "redis-db.com" domain name since
+> 2009! you can WHOIS it to check). So in my opinion calling the company
+> "RedisDB" is wrong, especially since I and Garantia Data from time to time
+> have friendly conversations as we are both part of the "Redis ecosystem", but
+> I did not received any prior question about this issue.
+
+The following year, 2014, Garantia Data [changed its name to Redis Labs][redislabs].
+
+[redislabs]: https://techcrunch.com/2014/01/29/database-provider-garantia-data-makes-another-name-change-this-time-to-redis-labs/)
+
+In 2015, Salvatore left Pivotal for Redis Labs. He writes in his blog post
+[Thanks Pivotal, Hello Redis Labs](http://antirez.com/news/91):
+
+> Redis Labs was willing to continue what VMware and Pivotal started. I'll be
+> able to work as I do currently, spending all my time in the open source side
+> of the project, while Redis Labs continues to provide Redis users with an
+> hassles-free Redis experience of managed instances and products.
+
+In 2018, Redis Labs changed the license of some of its modules from AGPL to a
+source-available license. The license prevents competing cloud providers from
+offering these modules to customers and does therefore not fulfil the creteria
+for an open source license.
+
+This was interpreted by some as if Redis is no longer open source. Sanfilippo
+clarified on his blog [Redis will remain BSD licensed][antirez-remain-bsd].
+
+Yiftach Shoolman, CTO and co-founder of Redis Labs, also clarified in in the
+company's blog that [Redis’ License is BSD and will remain
+BSD][will-remain-bsd]. He repeated this promise in [a comment on Hacker
+News][hn-always-bsd], writing “let me assure you that Redis remains and always
+will remain, open source, BSD license”.
+
+[antirez-remain-bsd]: http://antirez.com/news/120
+[will-remain-bsd]: https://redis.io/blog/redis-license-bsd-will-remain-bsd/
+[hn-always-bsd]: https://news.ycombinator.com/item?id=17819392
+
+Also in 2018, Redis Labs aquired the trademark rights to the name Redis and the
+logo from Sanfilippo.
+
+In 2020, Salvatore Sanfilippo announced in his blog post [The end of the Redis
+adventure](http://antirez.com/news/133) that he is stepped back as the Redis
+maintainer, handing over the maintenance to Yossi Gottlieb and Oran Agra at
+Redis Labs. The two created a “core team” to maintain the project and invited
+Itamar Haber from Redis Labs, Zhao Zhao from Alibaba and Madelyn Olson from
+Amazon. The members were selected “based on demonstrated, long-term personal
+involvement and contributions”. This was described in the projects
+[Governance][governance-2020] page which was the inspiration for the current
+[Valkey governance](valkey-governance).
+
+[governance-2020]: https://web.archive.org/web/20200709170526/https://redis.io/topics/governance
+[valkey-governance]: https://github.com/valkey-io/valkey/blob/unstable/GOVERNANCE.md
+
+In 2021, the Redis Labs changed its name to Redis.
```

Comment:
If you want avoid calling them Redis, you can call them Redis Inc. or Redis ltd. It's at least backed up by their wikipedia page, then you can continue to call the project Redis.

```suggestion
In 2021, the Redis Labs changed its name to Redis ltd.
```

Reply:
Yes. I somehow wanted to make a point of the confusion they caused by having the same name for two things. But you're right, I'll use Ltd. as in on wikipeda. (Is it registered in multiple countries, one Inc. and one Ltd.? This is unclear to me but perhaps it doesn't matter.)

Reply:
My vague understanding is that Redis Inc. is the top level parent company, but Redis ltd. owns all of the technology.